ReadMe.md

```diff
@@ -58,7 +58,10 @@ To edit | 編集（する） | へんしゅう |
 Automatic | 自動 | じどう |
 Screen | 画面 | がめん |
 Video | 動画 | どうが |
+Picture | 写真 |　しゃしん
+Download | ダウンロード
```

Comment:
The "To download" entry will suffice, we don't need to double all our suru-verbs, so this line you may remove.